obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  mag_aca: 11.406
+  mag_aca_err: 0.1
+- agasc_id: 974791856
+  mag_aca: 11.375
+  mag_aca_err: 0.1
+- agasc_id: 1020796600
+  mag_aca: 11.5
+  mag_aca_err: 0.1
+- agasc_id: 1058938112
+  mag_aca: 10.906
+  mag_aca_err: 0.1
+- agasc_id: 1081092600
+  mag_aca: 11.0
+  mag_aca_err: 0.1
+- agasc_id: 1092619768
+  mag_aca: 11.141
+  mag_aca_err: 0.1
+- agasc_id: 1105592952
+  mag_aca: 10.734
+  mag_aca_err: 0.1
+- agasc_id: 1108353064
+  mag_aca: 11.094
+  mag_aca_err: 0.1
+- agasc_id: 1139171624
+  mag_aca: 10.297
+  mag_aca_err: 0.1
+- agasc_id: 1143744224
+  mag_aca: 10.6
+  mag_aca_err: 0.1
+- agasc_id: 1178221128
+  mag_aca: 10.031
+  mag_aca_err: 0.1
+obs:
+- agasc_id:
+  - 901664112
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:129:13:39:37.186
+  status: 1
+- agasc_id:
+  - 686557584
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:221:17:58:51.959
+  status: 1
+- agasc_id:
+  - 897878136
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:228:21:05:34.379
+  status: 1
+- agasc_id:
+  - 857375576
+  comments: Expected star position is >1000 arcsec away during the whole obs
+  mp_starcat_time: 2003:357:02:49:14.852
+  status: 1
+- agasc_id:
+  - 857375848
+  comments: Expected star position is >1000 arcsec away during the whole obs
+  mp_starcat_time: 2003:357:02:49:14.852
+  status: 1
+- agasc_id:
+  - 188482256
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2004:055:07:41:03.254
+  status: 1
+- agasc_id:
+  - 654050016
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2004:109:04:09:26.608
+  status: 1
+- agasc_id:
+  - 920782072
+  comments: NSM at start of observation
+  mp_starcat_time: 2004:200:06:56:32.527
+  status: 1
+- agasc_id:
+  - 920783352
+  comments: NSM at start of observation
+  mp_starcat_time: 2004:200:06:56:32.527
+  status: 1
+- agasc_id:
+  - 897607632
+  comments: Large magnitude variation. Known eclipsing binary
+  mp_starcat_time: 2004:212:17:56:19.266
+  status: 1
+- agasc_id:
+  - 76813072
+  comments: mag 8.1 spoiler 15 arcsec away
+  mp_starcat_time: 2004:277:04:10:18.167
+  status: 1
+- agasc_id:
+  - 257953336
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:038:01:15:06.000
+  status: 1
+- agasc_id:
+  - 35263288
+  comments: Faint star with imposter. Magnitude is good
+  mp_starcat_time: 2005:056:20:11:01.041
+  status: 0
+- agasc_id:
+  - 723657296
+  comments: Faint star. Almost never tracked. Set mag to 11.547
+  mp_starcat_time: 2005:074:22:44:33.953
+  status: 1
+- agasc_id:
+  - 652480464
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:150:01:28:35.423
+  status: 1
+- agasc_id:
+  - 615914552
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2005:281:04:44:20.882
+  status: 1
+- agasc_id:
+  - 839385704
+  comments: Almost never acquired. When acquired, it was too bright. Star has been
+    observed other times. Known symbiotic star
```

Comment:
Unclear if we should take action to avoid using this as a guide star in the future (bad list or larger mag error).